vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1060,6 +1064,89 @@ def inplace_fused_experts_fake(
 )
 
 
+def flashinfer_fused_moe_fp8(router_logits: torch.Tensor,
+                             e_score_correction_bias: torch.Tensor,
```

Comment:
Use `flashinfer_fused_moe_blockscale_fp8` to differentiate between other moe variants in FI

Reply:
also add `assert fi_fused_moe is not None`

Reply:
Done. Thx.